src/store/modules/userInput.js

```diff
@@ -0,0 +1,44 @@
+const state = {
```

Comment:
Ten moduł VueX duplikuje aktualnie występujący stan w store:

https://github.com/fajnie-ze-wiesz/fzw-webapp/blob/d7fbb3a3f769f8567dd5c9c398e204994ead889e/src/store/index.js#L18-L19

Także widziałbym potencjalnie wyodrębnienia tych dwóch pól (`user` i `quizSetupInfo`) do osobnych modułów, natomiast myślę że lepiej zrobić to później (w kolejnym PR) i potencjalnie nawet post MVP jako że takie refactory mogą sporo namieszać przed wypuszczeniem MVP.